site/content/kapp/docs/develop/config.md

```diff
@@ -217,6 +222,16 @@ waitRules:
 
 `diffMaskRules` specify which field values should be masked in diff. By default `v1/Secret`'s `data` fields are masked. Currently only applied to `deploy` command.
 
+### preflightRules
+
+Available in v0.60.1+.
```

Comment:
We usually make a minor release when there are new features involved, so we will most likely have v0.61.0.

Reply:
Yeah, I wasn't sure on the version number (I guessed based on history). I can change it.